dev/tasks/tasks.yml

```diff
@@ -121,9 +121,9 @@ groups:
   ######################## Tasks to run regularly #############################
 
   nightly:
-    - debian-*-amd64
-    - ubuntu-*-amd64
-    - centos-*-amd64
+    - debian-*
```

Comment:
I managed to make the arm builds more stable on travis using graviton machines, so I think we should exercise these on a nightly basis.

Reply:
Is it OK that Ursa Computing keeps Travis CI subscription?